tiling.js

```diff
@@ -3797,20 +3816,35 @@ function cycleWindowWidth(metaWindow) {
     metaWindow.move_resize_frame(true, targetX, frame.y, targetWidth, frame.height);
 }
 
-function cycleWindowHeight(metaWindow) {
+function cycleWindowHeight(metawindow) {
+    return cycleWindowHeightDirection(metawindow, 0);
+}
+
+function cycleWindowHeightBackwards(metawindow) {
+    return cycleWindowHeightDirection(metawindow, -1);
+}
+
+function cycleWindowHeightDirection(metaWindow, direction) {
```

Comment:
For direction, consider using something more descriptive than numbers to signify direction.  You could use `Meta.MotionDirection.LEFT`, `Meta.MotionDirection.RIGHT` like we do with [`space.swap`](https://github.com/paperwm/PaperWM/blob/04871fc0b47e3ab385ae7c856d332bbeca2b09b3/tiling.js#L773), or you could create an enum for this like we do in `PreviewMode` or `FocusModes` see https://github.com/paperwm/PaperWM/blob/04871fc0b47e3ab385ae7c856d332bbeca2b09b3/tiling.js#L36